src/main/java/net/snowflake/client/jdbc/SnowflakeResultSetSerializableV1.java

```diff
@@ -903,18 +903,54 @@ private void setupTransientFields() throws SQLException {
    * ResultSet is sessionless.
    *
    * @return a ResultSet which represents for the data wrapped in the object
+   * @deprecated Please use new interface function getResultSet(ResultSetRetrieveConfig)
    */
+  @Deprecated
   public ResultSet getResultSet() throws SQLException {
```

Comment:
Is this a public API for the customer? If not, can we remove this?

Reply:
It's private API and getResultSet() is not used by SC any more. The only reason to keep it is that both both SC and JDBC are open source, I am afraid some customer may figure it and use it. I can remove it, if you think it is preferred.

Reply:
Let's remove unused code if it is not publicly documented.

Reply:
I will remove getResultSet() in next patch.
Do you think whether we should remove getResultSet(Properties)? getResultSet(Properties) is used by some released versions of SC. The only potential reason to keep it is that it can tolerate the use case that some user uses OLD SC to work with new JDBC version.

Reply:
ah, ok, then you may want to keep them unless Spark Connector sets the minimum supported JDBC version. You should consider it sooner or later otherwise we'll have to support unused code forever.

Reply:
I will add one more comment to indicate when it can be removed safely.